ddl/cluster.go

```
@@ -156,44 +237,53 @@ func (w *worker) onFlashbackCluster(d *ddlCtx, t *meta.Meta, job *model.Job) (ve
 		return ver, err
 	}
 
-	// stage 1, check and set FlashbackClusterJobID.
+	// stage 1, check and set FlashbackClusterJobID, save pd schedule.
 	if flashbackJobID == 0 {
 		err = kv.RunInNewTxn(w.ctx, w.store, true, func(ctx context.Context, txn kv.Transaction) error {
 			return meta.NewMeta(txn).SetFlashbackClusterJobID(job.ID)
 		})
 		if err != nil {
 			job.State = model.JobStateCancelled
```

Do we need to return an error here?

---

Updated, @zimulala